Resources/public/js/external/ez-alloyeditor.js

```diff
@@ -9,6 +9,4 @@ YUI.add('ez-alloyeditor', function (Y) {
     Y.namespace('eZ');
 
     Y.eZ.AlloyEditor = AlloyEditor;
-    Y.eZ.React = Y.eZ.AlloyEditor.React;
```

Comment:
removing those lines is BC break, so we need to keep the definition of `Y.eZ.React` and `Y.eZ.ReactDOM` somehow.